tokio-tls/Cargo.toml

```diff
@@ -35,6 +38,18 @@ cfg-if = "0.1"
 env_logger = { version = "0.6", default-features = false }
 futures = { version = "0.3.0", features = ["async-await"] }
 
+[features]
+tokio-nativetls = [
```

Comment:
The feature flag names can just be the names of the impls (`native-tls` and `rustls`).

Reply:
Done!

Reply:
Done!